src/cmd-buildupload

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python3
+
+# The inverse of cmd-buildprep.
+
+import argparse
+import json
+import os
+import subprocess
+import sys
+import tempfile
+
+sys.path.insert(0, '/usr/lib/coreos-assembler')
+from cmdlib import load_json  # noqa: E402
+
+
+def main():
+    args = parse_args()
+    args.func(args)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--build", help="Build ID", default='latest')
```

Comment:
Maybe `--id` or `--build-id`?

Reply:
I agree `--build-id` is likely a better name, though... this is matching the switch name in all the other commands. Guess we could rename them all and alias for a bit, though not sure if it's worth it. :)

Reply:
Ah that's a good point; I just didn't think `buildupload --build <id>` was the most informative name; however, if it's matching the other commands I'm okay dropping the comment